pyhoma/client.py

```diff
@@ -21,6 +22,10 @@
 API_URL = "https://tahomalink.com/enduser-mobile-web/enduserAPI/"  # /doc for API doc
 
 
+async def relogin(invocation: Dict[str, Any]) -> None:
+    await invocation["args"][0].login()
```

Comment:
What is the difference between this and just calling `login`?

Reply:
backliff needs a callback method with a specific signature.

Reply:
Looks ugly but backoff library looks like a good solution 👍🏻